pycaret/internal/plots/time_series.py

```diff
@@ -1036,3 +1047,192 @@ def plot_time_series_decomposition(
 
     return fig, return_data_dict
 
+
+def plot_time_series_differences(
+    data: pd.Series,
+    model_name: Optional[str] = None,
+    data_kwargs: Optional[Dict] = None,
+    fig_kwargs: Optional[Dict] = None,
+):
+    fig, return_data_dict = None, None
+
+    if data_kwargs is None:
+        data_kwargs = {}
+    if fig_kwargs is None:
+        fig_kwargs = {}
```

Comment:
This is completely fine, but this would also work:
```suggestion
    data_kwargs = data_kwargs or {}
    fig_kwargs = fig_kwargs or {}
```